data_retrieval/pgp_harvard.py

```diff
@@ -168,12 +159,99 @@ def parse_pgp_profile_page(huID):
     return genome_file_links, surveys
 
 
-def create_pgpharvard_ohdatasets(huID,
-                                 task_id=None,
-                                 update_url=None,
-                                 **kwargs):
+def vcf_from_var(vcf_filename, tempdir, var_filepath):
+    data_files = []
```

Comment:
Unnecessary. Add a comment "returns an array of X" and the array can be defined with a single item later.